### PR TITLE
Add service-serving-cert-signer image

### PR DESCRIPTION
**Pull request by smarterclayton**
@deads2k

### CODE REVIEW

File: ci-operator/config/openshift/service-serving-cert-signer/master.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    {
+      "as": "unit",
+      "from": "src",
+      "commands": "ARTIFACT_DIR=/tmp/artifacts JUNIT_REPORT=1 make test-unit"
+    }
+  ],
+
+  "resources": {
+    "*": {
+      "requests": { "cpu": "100m", "memory": "200Mi" },
```
Comment (deads2k) [inline ci-operator/config/openshift/service-serving-cert-signer/master.json]: more power!

File: ci-operator/config/openshift/telemeter/master.json [hunk 2]
```diff
@@ -18,15 +18,18 @@
     "cluster": "https://api.ci.openshift.org",
     "namespace": "openshift",
     "name": "release",
-    "tag": "golang-1.9"
+    "tag": "golang-1.10"
```
Comment (deads2k) [inline ci-operator/config/openshift/telemeter/master.json]: this doesn't have to match what's in the docker file?  I thought that dockerfile said 1.9

File: ci-operator/config/openshift/service-serving-cert-signer/master.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    {
+      "from": "base",
+      "to": "service-serving-cert-signer",
+      "inputs": {
+        "root": {"as": ["0"]}
+      }
+    }
+  ],
+
+  "tests": [
```
Comment (deads2k) [inline ci-operator/config/openshift/service-serving-cert-signer/master.json]: will I get a test that tries to build the image too or is that only done when master moves?

File: cluster/ci/config/prow/config.yaml [hunk 4]
```diff
@@ -1075,6 +1076,35 @@ presubmits:
         - --target=unit
         - --target=[images]
 
+  openshift/service-serving-cert-signer:
+  - name: pull-ci-openshift-service-serving-cert-signer-unit
+    agent: kubernetes
```
Comment (deads2k) [inline cluster/ci/config/prow/config.yaml]: what is this?

File: ci-operator/config/openshift/telemeter/master.json (hunk 2 above)
Comment (smarterclayton) [reply ci-operator/config/openshift/telemeter/master.json]: it does not, i put you on 1.10 just to start.  This is effectively the "official" ci run

File: ci-operator/config/openshift/service-serving-cert-signer/master.json (hunk 3 above)
Comment (smarterclayton) [reply ci-operator/config/openshift/service-serving-cert-signer/master.json]: the unit test will also build the image, if either fails it'll fail

File: cluster/ci/config/prow/config.yaml (hunk 4 above)
Comment (smarterclayton) [reply cluster/ci/config/prow/config.yaml]: agent?  means prow launches a pod (the spec below) that has your stuff

**Comment by deads2k** [comment]
just some questions to try to understand roughly what this thing is doing.

**Pull request merged**